packages/@coorpacademy-components/src/organism/review-slide/index.js

```diff
@@ -151,22 +154,28 @@ const Slide = (props, context) => {
 
   return (
     <div data-name={`slide-container`} className={style.slide}>
-      <QuestionContainer
-        questionOrigin={parentContentTitle}
-        questionText={questionText}
-        answerUI={answerUI}
-      />
-      <ValidateButton
-        slideIndex={slideIndex}
-        validateButton={validateButton}
-        primarySkinColor={primarySkinColor}
-      />
-      <CorrectionPopin
-        correctionPopinProps={correctionPopinProps}
-        slideIndex={slideIndex}
-        showCorrectionPopin={showCorrectionPopin}
-        animateCorrectionPopin={animateCorrectionPopin}
-      />
+      {loading ? (
+        <Loader className={style.loader} theme="default" aria-label={loadingAriaLabel} />
+      ) : (
+        <>
+          <QuestionContainer
+            questionOrigin={parentContentTitle}
+            questionText={questionText}
+            answerUI={answerUI}
+          />
+          <ValidateButton
+            slideIndex={slideIndex}
+            validateButton={validateButton}
+            primarySkinColor={primarySkinColor}
+          />
+          <CorrectionPopin
+            correctionPopinProps={correctionPopinProps}
+            slideIndex={slideIndex}
+            showCorrectionPopin={showCorrectionPopin}
+            animateCorrectionPopin={animateCorrectionPopin}
+          />
+        </>
```

Comment:
est-ce que tu peux modifier le `<> `et utiliser plutôt un `div`?
```suggestion
        <div>
          <QuestionContainer
            questionOrigin={parentContentTitle}
            questionText={questionText}
            answerUI={answerUI}
          />
          <ValidateButton
            slideIndex={slideIndex}
            validateButton={validateButton}
            primarySkinColor={primarySkinColor}
          />
          <CorrectionPopin
            correctionPopinProps={correctionPopinProps}
            slideIndex={slideIndex}
            showCorrectionPopin={showCorrectionPopin}
            animateCorrectionPopin={animateCorrectionPopin}
          />
        <div/>
```